### PR TITLE
#10  View Trainee Information

### LINKED ISSUE #8: View info about a trainee
Please check this link for figma https://www.figma.com/design/Y6yBiYmYqJvlKhf37eOqoo/Igor-steppers?node-id=1140-4663&t=yVDm5g9nXfAANFpO-4

### PR DESCRIPTION
**Pull request by aimeudoxie**
### What does this PR do?
This PR includes displaying of the trainee's profile information.

### Description of Task to be completed?
Once a trainee logs in they can be able to see their profile information such as Full Names, Email, Phone Number, GitHub Username, Resume, Biography.

### How should this be manually tested?
Open the app and click on Get started
Login as a trainee
click the top right profile Image to get trainee's profile information

### Screenshots
IMG_0818
IMG_0819

### CODE REVIEW

**Review by MugemaneBertin2001 — CHANGES_REQUESTED** [review]
It is too early to have this.
Screenshot 2024-10-09 162111

**Review by niyibi250 — CHANGES_REQUESTED** [review]
try to have design similar to figma design:

https://www.figma.com/design/Y6yBiYmYqJvlKhf37eOqoo/Igor-steppers?node-id=1142-4708&t=UcKcpSVBdUoqn1Tq-4

**Review by MugemaneBertin2001 — APPROVED** [review]

**Review by pacifiquemboni — APPROVED** [review]

**Review by Mbabazi12 — APPROVED** [review]

File: app/auth/login/index.tsx
Comment (ncutixavier) [inline app/auth/login/index.tsx:1]: I can see many changes done in this file. Do we need it? If no, remove them please.

File: assets/images/andela.jpeg
Comment (ncutixavier) [inline assets/images/andela.jpeg:1]: Where are you using this images?

File: assets/images/cover.jpg
Comment (ncutixavier) [inline assets/images/cover.jpg:1]: Where are you using this images?

File: assets/images/pp.jpg
Comment (ncutixavier) [inline assets/images/pp.jpg:1]: Where are you using this images?

File: components/sprintRatings.tsx
Comment (ncutixavier) [inline components/sprintRatings.tsx:1]: Please avoid unnecessary changes, they might cause conflicts

File: app/auth/login/index.tsx
Comment (aimeudoxie) [reply app/auth/login/index.tsx:1]: Yes we need them there was some errors which are fixed in this branch

File: assets/images/andela.jpeg
Comment (aimeudoxie) [reply assets/images/andela.jpeg:1]: Removed it

File: assets/images/pp.jpg
Comment (aimeudoxie) [reply assets/images/pp.jpg:1]: Done!
Removed it

File: assets/images/cover.jpg
Comment (aimeudoxie) [reply assets/images/cover.jpg:1]: Removed it!

File: components/sprintRatings.tsx
Comment (aimeudoxie) [reply components/sprintRatings.tsx:1]: Done!

**Review by ncutixavier — COMMENTED** [review]
Please refactor the profile components Sync with @Mbabazi12

**Comment by aimeudoxie** [comment]
> Please refactor the profile components Sync with @Mbabazi12

File: assets/images/profile.png
Comment (ncutixavier) [inline assets/images/profile.png:1]: For the users who doesn't have profile picture, you can create an avatar using the name. for example 
image

File: components/About_trainee.tsx
Comment (ncutixavier) [inline components/About_trainee.tsx:1]: File naming for components is wrong: `AboutTrainee.tsx`

File: components/Trainee_org.tsx
Comment (ncutixavier) [inline components/Trainee_org.tsx:1]: File naming is wrong

Comment (ncutixavier) [reply components/Trainee_org.tsx:1]: Wrap those trainee components under the trainee folder 

`components/trainee/<file>.tsx`

**Review by ncutixavier — COMMENTED** [review]
Review yourself to make sure that you are not making unnecessary changes

File: components/trainee/About_trainee.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          <Text className={`${textColor} text-lg`}>
+            {profile.githubUsername ? profile.githubUsername : 'Unavailable'}
+          </Text>
+        </View>
+        <Text className={`${textColor} text-xl`}>RESUME</Text>
+        <Text className={`${textColor} text-md m-2`}>
+          {profile.resume ? profile.resume : 'No Resume uploaded yet'}
+        </Text>
+      </View>
+      <View className={`${bgColor} w-[100%] p-6 rounded-md mt-5`}>
```
Comment (niyibi250) [inline components/trainee/About_trainee.tsx]: here add same margin buttom like mb-14 , because on some devices like mine i can't scrolling down to the end:

image

but if you add mb-14 it look like this:

image

Now i can see the page is ended

File: components/trainee/Trainee_org.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		<View className="flex-row gap-5 m-1 ">
+		  <Text className={`${textColor} text-lg`}>Admin email:</Text>
+		  <Text className={`${textColor} text-lg`}>devpulse@proton.me</Text>
+		</View>
+		<View className="flex-row gap-5 m-1 ">
+		  <Text className={`${textColor} text-lg`}>Role:</Text>
+		  <Text className={`${textColor} text-lg`}>{profile.user?.role || 'Unavailable'}</Text>
+		</View>
+	  </View>
+	  <View className={`${bgColor} w-[100%] p-4 rounded-md mt-3 text-bold `}>
```
Comment (niyibi250) [inline components/trainee/Trainee_org.tsx]: also add margin button like mb-14, so the end of page

**Review by niyibi250 — CHANGES_REQUESTED** [review]

File: components/trainee/About_trainee.tsx (hunk 1 above)
Comment (aimeudoxie) [reply components/trainee/About_trainee.tsx]: Done!

**Review by niyibi250 — APPROVED** [review]

File: assets/images/profile.png
Comment (ncutixavier) [inline assets/images/profile.png:1]: Do we need this image? If not remove it

**Review by ncutixavier — APPROVED** [review]
